src/validation.cpp

```diff
@@ -4016,7 +4016,7 @@ bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& sta
     if (hash == Params().GetConsensus().hashGenesisBlock)
         return true;
 
-    if (block.nBits != GetNextWorkRequired(pindexPrev, block, consensusParams)) {
+    if (block.nBits != GetNextWorkRequired(pindexPrev, block, false, consensusParams)) {
         return state.DoS(100, error("%s : incorrect proof of work at %d", __func__, nHeight),
```

Comment:
return state.DoS(100, error("%s : incorrect proof of %s at %d", __func__, fProofOfStake ? "stake" : "work", nHeight),